services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslMinPortSpeed.groovy

```diff
@@ -0,0 +1,53 @@
+package org.openkilda.functionaltests.spec.links
+
+import static org.junit.Assume.assumeTrue
+import static org.openkilda.messaging.info.event.IslChangeType.DISCOVERED
+import static org.openkilda.messaging.info.event.IslChangeType.MOVED
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.testing.model.topology.TopologyDefinition.Isl
+
+import spock.lang.Narrative
+
+@Narrative("""Minimal port speed value is chosen for ISL capacity.
+Sometimes an ISL have different port speed on its edges.
+In that case, we need to set ISL capacity and all bandwidth parameters according to minimal speed value.
+Eg. 10G on one side, and 1G on another side, the ISL should have a 1G capacity.""")
+class IslMinPortSpeed extends BaseSpecification {
+    def "System is able to set min port speed for isl capacity"() {
```

Comment:
`System sets min port speed for isl capacity`
'able to set' means that it is not a mandatory action

Reply:
noted